### DISCUSSION TITLE
How to install OSPSuite-Full.9.1.3 in MacBook Pro

### DISCUSSION THREAD

**Discussion by praveenniper** [Open-Systems-Pharmacology/Forum#778] [Questions & Problems] [answered]
I am new to OSP and will appreciate it greatly if someone can help me install OSPSuite-Full.9.1.3 in my MacBook pro. 

Please accept thanks in advance for your help and time.

**Comment by StephanSchaller** [comment, marked as answer]
Unfortunately, OSP tools PK-Sim and MoBi are Windows-only software.

You can run the software in command-line (without graphical user interface) on Unix Systems but I believe that is not your intended use-scenario?

**Comment by andanteyang** [comment]
I installed a parallel desktop with Windows 10 on my Mac Pro. PK-sim runs smoothly on the virtual Windows.

**Comment by msevestre** [comment]
@praveenniper I was about to suggest the same as @andanteyang
